File: cp.js

function processData(input) {
    let inputs = "i am met Janice";
    let lists = inputs.split(" ");
    lists.forEach(e => {
        data = fetch(`http://130.185.118.141/tech_savishkaar?word=${e}`);
        console.log(data)
    })
    return 0;
}

// process.stdin.resume();
// process.stdin.setEncoding("ascii");
// _input = "";
// process.stdin.on("data", function (input) {
//     _input += input;
// });

// process.stdin.on("end", function () {
//     processData(_input);
// })
processData("I met Janice");

File: cp.py
import requests
import json

string = input()

output = ""
string = string.split(" ")
for i in string:
    url = f"http://130.185.118.141/tech_savishkaar?word={i}"
    response = requests.get(url)
    if(response.ok):
        data = json.loads(response.content)
        output += data["encoded"]   
    output+=" "
output = output.rstrip()
print(output)
     